IdentityServer/v6/BFF/SplitHosts/FrontendHost/wwwroot/session.js

```diff
@@ -0,0 +1,80 @@
+﻿const userUrl = "https://localhost:5010/bff/user";
+
+window.addEventListener("load", async () => {
+  var req = new Request(userUrl, {
+    headers: new Headers({
+      "X-CSRF": "1",
+    }),
+    credentials: "include",
+  });
+
+  var username = document.querySelector("#username");
+
+  try {
+    var resp = await fetch(req);
+    if (resp.ok) {
+      document.querySelector(".logged-in").classList.remove("hide");
+
+      let claims = await resp.json();
+      console.log("user claims", claims);
+
+      let logoutUrlClaim = claims.find(
+        (claim) => claim.type === "bff:logout_url"
+      );
+      if (logoutUrlClaim) {
+        document.getElementById("logoutLink").href =
+          "https://localhost:5010" +
+          logoutUrlClaim.value +
+          "&returnUrl=https://localhost:5011/";
```

Comment:
This interpolation only works if there are query parameters in the logoutUrlClaim. I think that is always the case - but is it possible that there wouldn't be a sid and thus no query parameter? I could go read the source and find out if that can ever happen, but either way, this code is coupled really tightly to the specifics of the logoutUrlClaim and I don't love that.